sshtunnel.py

```diff
@@ -1086,8 +1090,10 @@ def get_keys(logger=None, host_pkey_directories=None, allow_agent=False):
 
         paramiko_key_types = {'rsa': paramiko.RSAKey,
                               'dsa': paramiko.DSSKey,
-                              'ecdsa': paramiko.ECDSAKey,
-                              'ed25519': paramiko.Ed25519Key}
+                              'ecdsa': paramiko.ECDSAKey}
+        if hasattr(paramiko, 'Ed25519Key'):
+            # NOQA: new in paramiko>=2.2: http://docs.paramiko.org/en/stable/api/keys.html#module-paramiko.ed25519key
```

Comment:
dependency now is paramiko>2.7 so, is this redundant?

Reply:
yes, We will remove it in the future